packages/peregrine/lib/talons/AuthModal/useAuthModal.js

```diff
@@ -54,9 +56,13 @@ export const useAuthModal = props => {
     }, [showMyAccount]);
 
     const handleSignOut = useCallback(() => {
+        // After logout, reset the store to set the bearer token.
+        // https://www.apollographql.com/docs/react/networking/authentication/#reset-store-on-logout
+        apolloClient.resetStore();
```

Comment:
This is important -- you'll see other calls to this function in this PR. We should have been doing it whenever the login state changed to make sure that requests through apollo used the bearer token. You can see this "bug" in [develop](https://develop.pwa-venia.com/). Make sure you're logged out, then log in and without a reload perform some action that would call a graphql endpoint. You'll see that the `authorization: Bearer <token>` header is not present on the request.

After this change, it will be.